src/ubuntu/22.04/helix/amd64/Dockerfile

```diff
@@ -43,6 +43,33 @@ RUN ln -sf /usr/bin/python3 /usr/bin/python && \
     pip download --no-deps helix-scripts --index-url https://dnceng.pkgs.visualstudio.com/public/_packaging/helix-client-prod/pypi/simple && \
     pip install ./helix_scripts-*-py3-none-any.whl
 
+# build MsQuic as we don't have binary package.
+# Ubuntu 22  does not have OpenSSL 1.1.1 package and Quic releases currently do not work with OpenSsl3
+# support for OpenSSL3 is early progress in msquic/main
+RUN apt-get update && \
+    apt-get install -qq -y libnuma-dev liblttng-ust-dev ruby-dev && \
+    gem install fpm && \
+    cd /tmp && \
+    git clone --depth 1 --single-branch --branch main --recursive https://github.com/dotnet/msquic && \
+    cd msquic/src/msquic && \
+    cmake -B build/linux/x64_openssl3 \
+        -DQUIC_OUTPUT_DIR=/tmp/msquic/src/msquic/artifacts/bin/linux/x64_Release_openssl3 \
```

Comment:
Does this need to be a full path? Do we even need this? I.e. how (un)predictable the output path is?

Reply:
it seems to. I started with the relative path from `-B` but then the output was created under the ` build/linux/x64_openssl3` e.g. `build/linux/x64_openssl3/artifacts/bin/linux/x64_Release_openssl3`.